package-lock.json

```diff
@@ -33310,7 +33293,7 @@
         "rollup-plugin-sourcemaps": "^0.6.3",
         "source-map-loader": "^5.0.0",
         "storybook": "^8.0.4",
```

Comment:
Looks like the only thing preventing this PR from completing is a change file. I can push it through.

Though I'm a bit suspicious of the PR desription: it says it's updating both `storybook-addon-pseudo-states` and `rimraf` but I only see the former in the package files. I may create a new PR just for the storybook change and then re-trigger Renovate to see what it thinks still needs updating.

Reply:
I think we should avoid having these sit around since they cause build churn. The PR contents look good. I think we can verify the same test using the dependency dashboard after merge. So gonna update change file and push through.

Reply:
>I think we should avoid having these sit around since they cause build churn

Agreed! I started on it yesterday but hit a bunch of unexplained package-lock diffs and got hung up trying to figure them out and then went OOO. Good call to just complete this PR!